src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -149,6 +164,11 @@ internal static List<ReceiveChannel> GetConfiguredChannels(ReadOnlySettings sett
             return settings.TryGet(out List<ReceiveChannel> channels) ? channels : new List<ReceiveChannel>();
         }
 
+        internal static (string type, string address) GetReplyToAddress(ReadOnlySettings settings)
```

Comment:
will this be backported? Using tuples might be an issue when targeting the latest major release?

Reply:
Yes it will be backported to the current release branch, but not further than that. If it's an issue there, I'll switch to an explicit`Tuple<string, string>` or create an internal DTO or whatever works.